client/src/components/WizardStepperStep.vue

```diff
@@ -0,0 +1,124 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 (eventually AGPL-3.0) 2016-present Scille SAS -->
+
+<template>
+  <div
+    class="wizard-stepper-step"
+  >
+    <!-- default -->
+    <div
+      class="shape"
+      :class="getClass(status)"
+    >
+      <div
+        class="left-line"
+      />
+      <div class="circle">
+        <div
+          v-if="status === 'done'"
+          class="inner-circle-done"
+        />
+        <div
+          v-if="status === 'active'"
+          class="inner-circle-active"
+        />
+        <ion-icon
+          v-if="status === 'done'"
+          class="icon-checkmark"
+          :icon="checkmark"
+          size="default"
+        />
+      </div>
+      <div
+        class="right-line"
+      />
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+export enum StepStatus {
+  DEFAULT = 'default',
+  DONE = 'done',
+  ACTIVE = 'active',
+}
+</script>
+
+<script setup lang="ts">
+import { IonIcon } from '@ionic/vue';
+import { checkmark } from 'ionicons/icons';
+import { defineProps } from 'vue';
+
+defineProps<{
+  status: StepStatus,
+}>();
+
+function getClass(status: string): any {
```

Comment:
`status` is of type `StepStatus`, not string.

Returned value is of type `string`, not any (probably need to add a default `return '';` at the end of the function.